open-craft/problem-builder
# [BB-2320] Problem-builder-v4 merged in the master

Merged the problem-builder-v4 into master. Created another branch `problem-builder-v3`  to maintain v3 for those who need it. 

Test instructions:

* verify that tests in CI reliably pass

Reviewer:


Issue: [BB-2320](https://tasks.opencraft.com/browse/BB-2320)
